src/client/components/app/Transactions/DeployLineTx.tsx

```diff
@@ -213,25 +213,28 @@ export const DeployLineTx: FC<DeployLineProps> = (props) => {
         // displayGuidance={displaySourceGuidance}
       />
       {inputAddressWarning !== '' ? <div style={{ color: '#C3272B' }}>{inputAddressWarning}</div> : ''}
-      <TxTTLInput
+      <TxNumberInput
         headerText={t('components.transaction.deploy-line.select-ttl')}
-        inputText={t('components.transaction.deploy-line.time-to-live')}
-        inputError={false}
-        amount={timeToLive}
-        onAmountChange={onAmountChange}
+        inputLabel={t('components.transaction.deploy-line.time-to-live')}
+        inputAlign="right"
+        width={'sm'}
+        placeholder={'30%'}
```

Comment:
are you intending to default to 30%? That seems incorrect, especially given that the initial value displayed in the modal is 0%.

Reply:
actually this is TTL, not cratio or revenue-split, so I would expect it to just be 0.

Reply:
tested it successfully deploys line